model_utils/model.py

```diff
@@ -173,43 +173,19 @@ def infer_loss_batch(self, infer_data):
         # run inference
         return self._loss_inferer.infer(input=infer_data)
 
-    def infer_batch(self, infer_data, decoding_method, beam_alpha, beam_beta,
-                    beam_size, cutoff_prob, cutoff_top_n, vocab_list,
-                    language_model_path, num_processes, feeding_dict):
-        """Model inference. Infer the transcription for a batch of speech
-        utterances.
+    def infer_probs_batch(self, infer_data, feeding_dict):
```

Comment:
Please keep the naming consistent.  Like below `decode_batch_beam_search`, please change `infer_probs_batch` to `infer_batch_probs`.

Reply:
Done